community/releases/versions.md

```diff
@@ -5,6 +5,22 @@ sidebar:
   nav: community
 ---
 
+The Spinnaker releases listed below are top-level versions tying together each
+Spinnaker subcomponents' versions. These have been validated together in an 
+end-to-end integration test suite curated by the Spinnaker community, and
+represent a more maintainable, easy to upgrade Spinnaker. While 
+it's still possible to install each Spinnaker subcomponent's versions 
```

Comment:
Sorry merged before I saw your comment. 

The mention alludes to direct downloads (which users are currently doing).